src/test/java/org/takes/facets/auth/PsByFlagTest.java

```diff
@@ -23,34 +23,125 @@
  */
 package org.takes.facets.auth;
 
+import com.google.common.collect.ImmutableMap;
 import java.io.IOException;
+import nl.jqno.equalsverifier.EqualsVerifier;
+import nl.jqno.equalsverifier.Warning;
 import org.hamcrest.MatcherAssert;
 import org.hamcrest.Matchers;
+import org.junit.Before;
 import org.junit.Test;
+import org.mockito.Mock;
+import org.takes.Response;
 import org.takes.rq.RqFake;
+import org.takes.rs.RsWithBody;
+import org.takes.rs.RsWithStatus;
+import org.takes.rs.RsWithType;
 
 /**
  * Test case for {@link PsByFlag}.
+ *
  * @author Yegor Bugayenko (yegor@teamed.io)
  * @version $Id$
  * @since 0.10
  */
 public final class PsByFlagTest {
+    /**
+     * Testable PsByFlag object.
+     */
+    private static PsByFlag psbyflag;
```

Comment:
@longtimeago I've used static here due to ```transient``` fields ```flag``` and ```passes``` and you are right this is not necessary here, I will change it to ```transient```.